package/src/contexts/messageInputContext/hooks/useCreateMessageInputContext.ts

```diff
@@ -114,9 +114,7 @@ export const useCreateMessageInputContext = ({
   UploadProgressIndicator,
 }: MessageInputContextValue & Pick<ThreadContextValue, 'thread'>) => {
   const editingdep = editing?.id;
-  const fileUploadsValue = fileUploads
-    .map(({ duration, paused, progress, state }) => `${state},${paused},${progress},${duration}`)
-    .join();
+  const fileUploadsValue = fileUploads.map(({ state }) => state).join();
```

Comment:
Why are we getting rid of the other values here ? I feel like this could break memoization for file uploads, but not sure

Reply:
It was vague. I tested it after removal and we don't need it.